src/disco/metrics/generated/fd_metrics_gossip.c

```diff
@@ -0,0 +1,6 @@
+/* THIS FILE IS GENERATED BY gen_metrics.py. DO NOT HAND EDIT. */
```

Comment:
Should there be changes to `metrics.xml` that caused these generated files to change?

Reply:
yep sorry, I forgot to check this in

Reply:
Oh nvm... it wasn't showing up